commands/scanpullrequest.go

```diff
@@ -83,7 +83,9 @@ func scanPullRequest(params *utils.FrogbotParams, client vcsclient.VcsClient) er
 
 func getCommentFunctions(simplifiedOutput bool) (utils.GetTitleFunc, utils.GetSeverityTagFunc) {
 	if simplifiedOutput {
-		return utils.GetSimplifiedTitle, utils.GetEmojiSeverityTag
```

Comment:
Could it be that the `GetEmojiSeverityTag` function is no longer in yse following this change, except by the test?
If so, let's remove tis function and its test.

Reply:
done